Maui.DataGrid/DataGrid.xaml.cs

```diff
@@ -54,14 +54,19 @@ internal void SortItems(SortData sortData)
         var column = Columns[sortData.Index];
         var order = sortData.Order;
 
-        if (!IsSortable)
+        if (column.PropertyName == null)
         {
-            throw new InvalidOperationException("DataGrid is not sortable");
+            throw new InvalidOperationException($"Please set the {nameof(column.PropertyName)} of the column");

```

Comment:
why do you need to use `nameof `operator? Just `column.PropertyName` should be fine.

Reply:
I like to avoid the use of string literals where possible.

The strong type system of C# means that you can refactor without worrying about causing runtime errors as much.

If you make a habit of always using name of when referencing member names in literals, then you can use auto-renaming if you ever need to.

Also, nameof is computed at compile time, and the most recent versions of .NET will be able to do string interpolation and concatenation at compile time as well.

You would even be able to mark this string as const.